src/components/WwwFrame/WwwPage.vue

```diff
@@ -28,6 +28,9 @@ import TheFooter from './TheFooter';
 import TheBasketBar from './TheBasketBar';
 import TheBannerArea from './TheBannerArea';
 
+const hasLentBeforeCookie = 'kvu_lent_before';
+const hasDepositBeforeCookie = 'kvu_deposit_before';
```

Comment:
It's best to keep the cookie names short, like `kvu_lb` and `kvu_db`

Reply:
Got it, will change!